src/DynamoCore/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Greg" version="1.0.6176.18754" targetFramework="net45" />
+  <package id="Greg" version="1.1.6905.23659" targetFramework="net47" />
```

Comment:
Oh, I was not aware the .Net change is merged. How do we publish the nuget of Greg?

Reply:
its not merged, I pushed the nugets only to my local repo. I will push the nugets manually once this PR https://github.com/DynamoDS/PackageManagerClient/pull/20 gets LGTM

Reply:
the 47 here is automatically added - it is just the framework version of the project referencing this dll - it lets nuget know to prefer that version if it exists, but it doesnt exist.

Reply:
Make sense